frontend/src/components/roadmap/sections/RoadmapWhatWereWorkingOn.tsx

```diff
@@ -11,36 +11,31 @@ export default function RoadmapWhatWereWorkingOn() {
 
   return (
     <RoadmapPageSection className={"bg-white"} title={t("title")}>
-      <div className="margin-top-1" />
```

Comment:
- Remove unnecessary `div` (I think it's just there to add vertical space; other styles handle this)